### PR TITLE
Gui Rodrigues

### PR DESCRIPTION
**Pull request by qruz-hq**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,135 +20,514 @@
...
+                            <path
+                                d="M9.741.85a.75.75 0 01.375.65v13a.75.75 0 01-1.125.65l-6.925-4a3.642 3.642 0 01-1.33-4.967 3.639 3.639 0 011.33-1.332l6.925-4a.75.75 0 01.75 0zm-6.924 5.3a2.139 2.139 0 000 3.7l5.8 3.35V2.8l-5.8 3.35zm8.683 4.29V5.56a2.75 2.75 0 010 4.88z">
+                            </path>
+                            <path d="M11.5 13.614a5.752 5.752 0 000-11.228v1.55a4.252 4.252 0 010 8.127v1.55z"></path>
+                        </svg>
+                    </button>
+                    <div class=" relative">
+                        <div id="volume_line" data-position="50%">
+                            <span></span>
+                        </div>
```
Comment (qruz-hq) [inline index.html:527]: USE AND INPUT !!!!

File: css/style.css [hunk 2]
```diff
@@ -1,47 +1,377 @@
...
+.playing-option:hover {
+  fill: white;
+}
+
+.current-play-btn-container {
+    position: relative;
+    height: 45px;
+    width: 45px;
+    padding:0;
 }
```
[the remark this answers is not in the document]
Comment (Creatorrr1) [reply css/style.css:377]: You did make it easier for me to review since everything is all here

File: css/style.css [hunk 3]
```diff
@@ -1,47 +1,377 @@
...
+    border: none;
+    cursor: pointer;
+}
+
+section h2 {
+  font-weight: 550;
+  position: sticky;
+  left: 0;
+}
+
```
Comment (Creatorrr1) [inline css/style.css:20]: Good way of added an attribute to a specific class without effecting the other h2 tags.

File: css/style.css [hunk 4]
```diff
@@ -1,47 +1,377 @@
...
+#top-bar button:hover {
+    fill: white;
+}
 /* CARDS */
 
 .card {
-    text-align: center;
+  background-color: #171717;
+  transition: all var(--transition-timing) ease-out;
```
Comment (Creatorrr1) [inline css/style.css:189]: good way of implementing transitions effectively.

File: css/fluid.css [hunk 5]
```diff
@@ -0,0 +1,386 @@
...
+  height: 100vh;
+}
+
+.w-100v {
+  width: 100vw;
+}
+
+.w-auto {
+  width: auto;
+}
```
Comment (d-username) [inline css/fluid.css:34]: wooooow, this whole fluid.css is something totally opposite to what I did, but I kind of like it.

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,377 @@
...
+#volume_line {
+  background: gray;
+  border-radius: 5px;
+  overflow: hidden;
+  height: 5px;
+  margin: 0px 5px;
+}
+
+#player_line::after,
+#volume_line::after {
```
Comment (Alex90Jennings) [inline css/style.css:310]: what is ::after? I haven't seen it before

File: index.html [hunk 7]
```diff
@@ -1,12 +1,537 @@
...
+                                    <svg role="img" height="24" width="24" viewBox="0 0 24 24"
+                                        class="Svg-sc-1bi12j5-0 hDgDGI">
+                                        <path
+                                            d="M7.05 3.606l13.49 7.788a.7.7 0 010 1.212L7.05 20.394A.7.7 0 016 19.788V4.212a.7.7 0 011.05-.606z">
+                                        </path>
+                                    </svg>
+                                </button>
+                            </div>
+                            <p>Hot Hits BE</p>
+                            <small>50 Hits van dit moment</small>
```
Comment (Alex90Jennings) [inline index.html:333]: what is <small>?

**Pull request closed without merging**